### PR TITLE
Add markup and stylesheet for Homepage.

### PR DESCRIPTION
**Pull request by BenjaminSemah**
**I have added added the following;**

-Markup for the Homepage.
-Stylesheet for the Homepage.
-Main photo used for the tribute project.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    </div>
+
+    <div class="footer">
+        <p>You can read more about this Jomo kenyatta on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Jomo_Kenyatta">Wikipedia Entry</a></p>
+    </div>
+
+</section>
+
+</body>
+</html>
```
Comment (BenjaminSemah) [inline Index.html:51]: There is nothing wrong here. However, it is recommended that you end your HTML with a new line.

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute page</title>
+</head>
+<body>
+
```
Comment (BenjaminSemah) [inline Index.html:11]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <title>Tribute page</title>
+</head>
+<body>
+    
+<section id="main">
+    <header>
+        <h1 id="title">Jomo Kenyatta</h1>
+        <p>The first president of Kenya</p>
+    </header>
+
```
Comment (BenjaminSemah) [inline Index.html:17]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        <p>The first president of Kenya</p>
+    </header>
+    
+    <div id = "img-div">  
+        <figure>
+            <img class="responsive" src="jomoimage.jpg" alt="jomo kenyatta image" id="image">
+            <figcaption>Jomo Kenyatta was the first president of Kenya.</figcaption>
+        </figure>
+    
+
```
Comment (BenjaminSemah) [inline Index.html:24]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="quotation">
+        <blockquote>
+            Our children may learn about the heroes of the past. Our task is to make ourselves the architects of the future.
+        </blockquote>
+        <cite>
+            Jomo Kenyatta.
+        </cite>
+    </div>
+
```
Comment (BenjaminSemah) [inline Index.html:43]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    <div id="tribute-info">
+        <ul>
+            <li><strong>1987</strong>Jomo Kenyatta was born in the year 1987 in Ngindi, British East Africa.</li>
+            <li><strong>1964</strong>He became the first president of Kenya on December 12, 1964</li>
+            <li><strong>1978</strong>He passed away on August 22, 1978 at the age of 80.</li>
+        </ul>
+    </div>
+
```
Comment (BenjaminSemah) [inline Index.html:34]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    
+    <div id = "img-div">  
+        <figure>
+            <img class="responsive" src="jomoimage.jpg" alt="jomo kenyatta image" id="image">
+            <figcaption>Jomo Kenyatta was the first president of Kenya.</figcaption>
+        </figure>
+    
+
+    </div>
+
```
Comment (BenjaminSemah) [inline Index.html:26]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html (hunk 4 above)
Comment (BenjaminSemah) [inline Index.html:24]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        </blockquote>
+        <cite>
+            Jomo Kenyatta.
+        </cite>
+    </div>
+
+    <div class="footer">
+        <p>You can read more about this Jomo kenyatta on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Jomo_Kenyatta">Wikipedia Entry</a></p>
+    </div>
+
```
Comment (BenjaminSemah) [inline Index.html:47]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

File: Index.html [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+</section>
+
```
Comment (BenjaminSemah) [inline Index.html:49]: In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.

**Review by BenjaminSemah — COMMENTED** [review]
Hello @BenjaminSemah 

Great job with your project so far.

Good points.
- No linters error ✔️ 
- Proper GitHub flow ✔️ 

There are some issues that you need to work on.

Changes Required ♻️ 
_Check the comments under the review._

_Feel free to leave any comments or questions in the PR thread if something is not 100% clear._

**Review by rustam-tolipov — APPROVED** [review]
## Hi @BenjaminSemah   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**